lib/interceptor.js

```diff
@@ -263,13 +277,22 @@ var interceptor = {
       );
     }
 
+    function shouldExcludeRequestByUrl(url) {
+      return window[NAMESPACE].excludedUrls.find(function (regex) {
+        return url.match(new RegExp(regex));
```

Comment:
Do the strings need to be escaped? Can they be converted into regex in the call to excludeUrls, rather than here?

Reply:
By checking that test is a function, we can do both. If test is a function, we assume we have a regex, if it is not, we assume we have a string, and create a regex from it.

Reply:
When I tried passing a RegExp originally, when it arrived in the browser it was an empty object ```{}```. There is a property on RegExp, [source](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/RegExp/source), that I am using now to allow for strings or regex